Scripts/GenerateSampleViewSourceCode.swift

```diff
@@ -136,25 +137,42 @@ private let sampleStructs = sampleMetadata
     }
     .joined(separator: "\n")
 
-private let entries = sampleMetadata
+private let commonSamples = sampleMetadata
+    .filter { $0.category != "Augmented Reality" }
     .map { sample in "\(sample.structName)()" }
     .joined(separator: ",\n        ")
-private let arrayRepresentation = """
+
+private let iOSSamples = sampleMetadata
```

Comment:
Maybe call this `iOSSpecificSamples`, and/or add some comment for both variables on why we separate these samples from the common samples, so it's easier to reason about in the future.